README.md

```diff
@@ -13,6 +13,9 @@ The Puppet master is configured under Apache and Passenger by default, unless
 3.x based installation, `server_implementation` can be set to `puppetserver`
 to switch to the JVM-based Puppet Server.
 
+When using Puppet Server 2 (version 2.0 was the first version to support Puppet 4),
+Puppet Server 2.2 is the minimum version supported by this module.
```

Comment:
@ekohl Sorry, I'm not sure I follow.  Which parameter? 'server_use_legacy_auth_conf'?